src/styles/themes/_markdown-theme.scss

```diff
@@ -2,81 +2,77 @@
 // This theme matches the syntax highlighting in VS Code
 
 markdown {
-  pre[class*="language-"],
-  code[class*="language-"] {
+  pre[class*='language-'],
+  code[class*='language-'] {
     color: #d4d4d4;
     font-size: 13px;
-    text-shadow: none;
-    font-family: Menlo, Monaco, Consolas, "Andale Mono", "Ubuntu Mono", "Courier New", monospace;
-    direction: ltr;
+    line-height: 1.5;
+    font-family: 'Menlo', 'Monaco', 'Consolas', 'Andale Mono', 'Ubuntu Mono', 'Courier New', monospace;
```

Comment:
do we need to disable the no font family, hex, or other rules here?

Reply:
Disabled stylelint for this file as its a 3rd party theme